bot/pagination.py

```diff
@@ -100,39 +89,24 @@ def add_line(self, line='', *, empty=False):
     async def paginate(cls, lines: Iterable[str], ctx: Context, embed: Embed,
                        prefix: str = "", suffix: str = "", max_lines: Optional[int] = None, max_size: int = 500,
                        empty: bool = True, restrict_to_user: User = None, timeout: int = 300,
-                       footer_text: str = None, url: str = None, exception_on_empty_embed: bool = False):
+                       footer_text: str = None, url: str = None, exception_on_empty_embed: bool = False) -> None:
```

Comment:
This functions has the `return await ctx.send` pattern in a couple of its paths, so this should be a `Optional[Message]` return annotation. E.g., see [line 165](https://github.com/python-discord/bot/blob/138567f371fa2db8675c2b5661ff5526e774335a/bot/pagination.py#L165).

I can't suggest it properly, though, since it would bring it over the line limit and the imports needs to be adjusted.(`Message` should be added to the `from discord import` line.)